src/fullpage/fullpage.tex

```diff
@@ -119,7 +120,7 @@
 	\foreach \row in {0,...,2} {
 		\node [below right] at (\LEFTMARGIN + 0.12in + \col * \CARDWIDTH, \TOPMARGIN + 0.12in + \row * \CARDHEIGHT) {\fontsize{20}{20}\selectfont Cards};
 		\node [below right] at (\LEFTMARGIN + 0.12in + \col * \CARDWIDTH, \TOPMARGIN + 0.42in + \row * \CARDHEIGHT) {\fontsize{20}{20}\selectfont  Against};
-		\node [below right] at (\LEFTMARGIN + 0.12in + \col * \CARDWIDTH, \TOPMARGIN + 0.72in + \row * \CARDHEIGHT) {\fontsize{20}{20}\selectfont  Cryptography};
+		\node [below right] at (\LEFTMARGIN + 0.12in + \col * \CARDWIDTH, \TOPMARGIN + 0.72in + \row * \CARDHEIGHT) {\fontsize{20}{20}\selectfont  Blockchain};
```

Comment:
Aren't these card backs?  You'd think the cards should mesh with other decks, but CAC already broke that here, so whatever.  I guess someone could change this back if they need it to mesh.

Reply:
Yes, I thought the same. Maybe makes sense to put this into README.

Reply:
We should figure out what we what here as the default regardless.